x/dispatch/dispatch.go

```diff
@@ -0,0 +1,20 @@
+package dispatch
+
+import (
+	"github.com/pokt-network/pocket-core/x/blockchain"
+	"github.com/pokt-network/pocket-core/x/session"
+)
+
+func DispatchPeers() {
+	// TODO returns tendermint peers cross checked with world state
+}
+
+// essentially a wrapper for session.NewSession()
+func DispatchSession(application session.SessionAppPubKey, nonNativeChain session.SessionBlockchain) (*session.Session, error) {
```

Comment:
Let's create an issue here to cache sessions so we don't have to generate them

Reply:
You had told me that caching was the job of infra. Have you changed your position on this?

Reply:
You are 100% right, what we need to figure out is how the caching will happen.

Reply:
Awesome, I'll create the issue